hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -275,6 +295,14 @@ public int getDeadNodeTimeout() {
     return getConf().getInt("mesos.hdfs.deadnode.timeout.seconds", DEFAULT_DEADNODE_TIMEOUT);
   }
 
+  public int getDFSReplication() {
+    String dfsReplicationString = System.getProperty("dfs.replication");
```

Comment:
We should be following the same format as the other config values here instead of using System.getProperty